src/Hadolint/Rules.hs

```diff
@@ -757,6 +758,16 @@ useJsonArgs = instructionRule code severity message check
     check (Entrypoint (ArgumentsText _)) = False
     check _ = True
 
+noApt :: Rule
+noApt = instructionRule code severity message check
+  where
+    code = "DL3026"
```

Comment:
This code is repeated, use the next number please

Reply:
oops, sorry about that, changed